src/operator/config/crd/k8s.otterize.com_clientintents.yaml

```diff
@@ -111,6 +111,9 @@ spec:
             type: object
           status:
             description: IntentsStatus defines the observed state of ClientIntents
+            properties:
+              hasSharedServiceAccount:
```

Comment:
A few notes here:
1. The status field is supposed to be user-facing, while this is used internally.
2. We discussed using the status field to reflect this information, and if we do this, we need to also reflect the other kinds of statuses here. This PR doesn't do that, so this attribute is the odd one out (everything is reflected through events)
3. Adding this to the schema will make it harder to change later, as it requires updating the CRD.

As this might be a temporary solution, let's please use an annotation instead, which keeps it as a pure implementation detail - we could change how this is implemented in the intents operator without any user-facing effect.